### ISSUE TITLE
performance issue

### ISSUE THREAD

**Issue by soans1994** [cybaj/fcn_angio#1]
hello author, 

i tried similar implementation, but the fcn 32 works better than fcn 16. did you get better results?

thank you

**Comment by cybaj** [comment]
No, I did not use fcn 32. But as you say, it will be better for switching the feature size.

But in that days, I choose to experiment with deeplab families or other segmentation models.
